### PR TITLE
feat: Android support

### PR DESCRIPTION
**Pull request by karasusan**
- Add Android platform to testing platforms on CI
- Add `Unity2020.3` into the editor list on CI
- Upgrade WebRTC package version `2.4.0-exp.1`

### CODE REVIEW

File: RenderStreaming~/Packages/packages-lock.json [hunk 1]
```diff
@@ -79,15 +97,68 @@
...
+      "version": "2.4.0-exp.1",
+      "depth": 0,
       "source": "registry",
       "dependencies": {
         "com.unity.modules.jsonserialize": "1.0.0",
         "com.unity.editorcoroutines": "1.0.0"
       },
       "url": "https://packages.unity.com"
     },
+    "com.unity.xr.arcore": {
```
Comment (kannan-xiao4) [inline RenderStreaming~/Packages/packages-lock.json:109]: Do we include the packages for these AR samples on init?

Comment (karasusan) [reply RenderStreaming~/Packages/packages-lock.json]: Not needed. Removed dependencies.

File: com.unity.renderstreaming/Runtime/Scripts/StreamReceiverBase.cs [hunk 2]
```diff
@@ -23,8 +23,14 @@ public abstract class StreamReceiverBase : MonoBehaviour, IStreamReceiver
...
-        /// <param name="track"></param>
-        public virtual void SetReceiver(string connectionId, RTCRtpReceiver receiver)
+        public virtual TrackKind Kind { get; }
+
+
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name="track"></param>
+    public virtual void SetReceiver(string connectionId, RTCRtpReceiver receiver)
```
Comment (kannan-xiao4) [inline com.unity.renderstreaming/Runtime/Scripts/StreamReceiverBase.cs:33]: [nits] indent

File: com.unity.renderstreaming/Tests/Runtime/SignalingHandlerTest.cs [hunk 3]
```diff
@@ -312,6 +307,7 @@ public IEnumerator ReceiveDataChannel()
             container1.test.component.AddComponent(channel1);
             container1.test.component.CreateConnection(connectionId);
             yield return new WaitUntil(() => container1.test.component.ExistConnection(connectionId));
+            Debug.Log("1");
```
Comment (kannan-xiao4) [inline com.unity.renderstreaming/Tests/Runtime/SignalingHandlerTest.cs:310]: [nits] no need logs

**Review by kannan-xiao4 — APPROVED** [review]
lgtm

**Pull request merged**